### ISSUE TITLE
Simplify installation

### ISSUE THREAD

**Issue by juanmc2005** [juanmc2005/StreamingSpeakerDiarization#93]
## Problem

Installation requires multiple steps, which were necessary when pyannote.audio 2.0 hadn't been released.
Now that this isn't needed anymore, it would be better to simplify the steps for a default installation including pyannote.audio even though it's optional.

Ideally this should be a single step: `pip install diart`

**Comment by hbredin** [comment]
Or 

```
pip install diart # does not install pyannote
pip install diart[pretrained] # installs pyannote
```

**Comment by hbredin** [comment]
In case you are wondering how this is done (I am sure you know already :)):

https://github.com/pyannote/pyannote-core/blob/5399294a65e6ad81ae4da5a906d84d515b832822/setup.py#L67-L78

**Comment by juanmc2005** [comment]
**Update:** pyannote.audio and pytorch installations already automated in v0.6

I prefer to leave a pyannote installation by default, advanced users that want to free space may do so by uninstalling it. It hasn't proven to be a required scenario for now.

The next step would be to automate portaudio and pysoundfile with pip if possible.

**Comment by juanmc2005** [comment]
Implemented in #130
